### PR TITLE
add BTC jam (BTC Lending site)

### PR DESCRIPTION
**Pull request by mikeatlas**

### CODE REVIEW

**Comment by mikeatlas** [comment]
only didnt pass Travis due to timeout of travis....

**Comment by Carlgo11** [comment]
it looks like you added 2 images. One in the PNG format and one in JPEG.

**Comment by mikeatlas** [comment]
@Carlgo11 you're right, there's only one jpg now 32x32 in the PR.

**Comment by Carlgo11** [comment]
although we prefer png

File: _data/bitcoin.yml [hunk 1]
```diff
@@ -60,6 +60,14 @@ websites:
       software: Yes
       doc: https://support.btc-e.com/index.php?/Knowledgebase/Article/View/97/25/how-do-i-set-2-factor-authentication-to-my-btc-e-account
 
+    - name: BTCJam
+      url: https://www.btcjam.com
+      img: btcjam.png
+      tfa: Yes
+      sms: Yes
```
Comment (Carlgo11) [inline _data/bitcoin.yml:67]: I don't see anything about sms more than that they verify your phone number upon initial TFA setup.

**Comment by mxxcon** [comment]
@mikeatlas at 32x32 PNG is 99% of the time smaller than jpg.
Your image is jpg, but your code refers to png. Please make it consistent.

**Comment by mikeatlas** [comment]
I give up trying to help contribute with PRs when you spend more time nitpicking me than accepting them and making tiny tweaks yourself. Appreciate your project immensely but I'll opt out of helping further.

FWIW, here's the same png/jpg, I ran the png version through [ImageOptim](https://imageoptim.com/) I went from 1kb JPG to 2kb PNG. So I opted for jpeg. But messed up between push/pull PRs in the interim, my apologies.
btcjam
btcjam

**Comment by mxxcon** [comment]
@mikeatlas We are not nitpicking, just making sure that data is accurate :)

**Comment by Carlgo11** [comment]
That picture doesn't look like any logo they use. Their favicon is favicon.ico and while googling for bitjam I see this logo google play bitjam logo

**Comment by mikeatlas** [comment]
I regret trying to help out. Their favicon is only 16x16 and you require 32x32, I picked a logo that looks like what their site uses right now. You can disregard this PR.

**Comment by Carlgo11** [comment]
While we do appreciate all contributions we need to do our job.
Our job is to make sure things follow the [guidelines](https://github.com/jdavis/twofactorauth/blob/master/CONTRIBUTING.md) of this repo before we can merge anything.

It sometimes is a lengthy process for a PR to follow the guidelines but all 703 PRs that have been merged have gone through it.

**Comment by mikeatlas** [comment]
I've seen your team fix "volunteered" PRs in the past before, instead of just nitpicking submissions

Example: https://github.com/jdavis/twofactorauth/pull/894

Good day.

**Comment by Carlgo11** [comment]
I don't really see what you're pointing at?
Josh is the creator of this repo and knows best of all what our guidelines are. Even so, a contributor did point out an error he made with formatting.

**Comment by jamcat22** [comment]
@mikeatlas Sorry about the trouble with your PR.

I'm in the process of rewriting the Contributing.md file to make it more friendly and easier to understand. If you have any suggestions of parts that were particularly confusing, feel free to tell me!

Sometimes we help out with PRs when people are having trouble with it or are new to GitHub. If that's the case, you can just tell us, and one of us can help. I think the reason we didn't do this in your case is that it seemed like you knew what you were doing.

If you don't want to make a new pr, I can take add it to my backlog of sites.

Thanks!

**Comment by mikeatlas** [comment]
Please do add it to your backlog. http://help.btcjam.com/customer/portal/articles/1987686 

I'm not forking a new repo, creating a branch, checking in changes, creating and submitting a PR for your project again when you could have easily accepted this and my other PR by dotting a missed "i" or crossing a missed "t" in about 2 minutes and had them up on your site by now. Instead you annoyed me and made me regret putting in any effort in the first place, sending me back and forth with pedantic rejections.

**Comment by jamcat22** [comment]
No problem.

I will talk to the contributors about fixing little things after they're merged instead of while the PR is opened.

I'll also make sure to be more descriptive in the Contributing.md and the wiki to make sure that there's nothing to fix in the first place. (Which was obviously a shortcoming on my part. :smile:)

Again, I'm sorry about how this PR went and I hope you consider contributing again.

**Comment by mikeatlas** [comment]
my logo choice was fine by the way. 

http://blog.btcjam.com/2015/05/07/how-to-use-authy-two-factor-authentication-with-btcjam/

**Pull request closed without merging**